### ISSUE TITLE
make install fails on mac os due to unsupported date command flags

### ISSUE THREAD

**Issue by pbochynski** [gardener/component-cli#79]
**What happened**:
`make install` reports a failure:
```
date: illegal option -- -
usage: date [-jnRu] [-r seconds|file] [-v[+|-]val[ymwdHMS]]
            [-I[date | hours | minutes | seconds]]
            [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]
```
**What you expected to happen**:
Successful installation

**How to reproduce it (as minimally and precisely as possible)**:
```
git clone git@github.com:gardener/component-cli.git
cd component-cli
make install
```

**Anything else we need to know**:

**Environment**:
macOS 12.3.1

**Comment by jschicktanz** [comment]
hi, this issue should be fixed by #81. Can you check if it is working for you now?

**Comment by pbochynski** [comment]
All good. Thanks!
